src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/msg/XIncludeMessages.properties

```diff
@@ -17,7 +17,7 @@
 
 # Messages for message reporting
 BadMessageKey = The error message corresponding to the message key can not be found.
-FormatFailed = An internal error occurred while formatting the following message:\n  
+FormatFailed = An internal error occurred while formatting the following message:\n
```

Comment:
At first glance, it might look like something should follow the :, but note that there is a \n so if anything this will only make the next line improperly indented.